test/shared/plugins/reset-password/controller.js

```diff
@@ -0,0 +1,288 @@
+'use strict';
+
+const Lab = require('lab');
+const {
+  experiment,
+  test,
+  beforeEach,
+  afterEach
+} = exports.lab = Lab.script();
+const { expect } = require('code');
+const sinon = require('sinon');
+const sandbox = sinon.createSandbox();
+const { set } = require('lodash');
+
+const loginHelpers = require('../../../../src/external/lib/login-helpers');
+const controller = require('../../../../src/shared/plugins/reset-password/controller');
+
+/**
+ * Note current version test of 'code' can't test for async function
+ * @see {@link https://github.com/hapijs/code/issues/103}
+ */
+function isAsync (fn) {
+  return fn.constructor.name === 'AsyncFunction';
+}
+
+experiment('reset password controller', () => {
+  experiment('Check methods on reset password controller', () => {
```

Comment:
I don't think we need these tests as the tests are exercised in the experiments below.